app/editor/src/components/layout/DefaultLayout.tsx

```diff
@@ -148,13 +150,22 @@ const DefaultLayout: React.FC<ILayoutProps> = ({
           </LayoutErrorBoundary>
         </div>
       </Show>
-      <Show visible={!keycloak.authenticated}>
+      <Show visible={!keycloak.authenticated && !isReportInstanceView}>
         <div className="main-window">
           <main style={{ backgroundColor: '#f2f2f2' }}>
             <UnauthenticatedHome />
           </main>
         </div>
       </Show>
+      <Show visible={!keycloak.authenticated && isReportInstanceView}>
+        <div className="main-window">
+          <LayoutErrorBoundary>
+            <main>
+              <Outlet />
```

Comment:
If it's an unauthenticated attempt to view the reports then redirect to subscriber site.  This is part of a temporary work around.